library/Rules/BankAccount.php

```diff
@@ -0,0 +1,59 @@
+<?php
+namespace Respect\Validation\Rules;
+
+use malkusch\bav\BAV;
+use Respect\Validation\Exceptions\ComponentException;
+
+/**
+ * Validates a bank account for a given bank.
+ * 
+ * Currently only German validation is supported (country code "de").
+ * This validator depends on the composer package malkusch/bav.
+ * 
+ * @author Markus Malkusch <markus@malkusch.de>
+ * @see BAV::isValidBankAccount()
+ */
+class BankAccount extends AbstractRule
+{
+
+    /**
+     * @var string
+     */
+    private $bank;
+    
+    /**
+     * @var String $countryCode The ISO 639-1 country code.
+     */
+    private $countryCode;
+    
+    /**
+     * Sets the country code and bank.
+     * 
+     * The country code is not case sensitive.
+     * 
+     * @param string $countryCode The ISO 639-1 country code.
+     * @param string $bank The bank.
+     */
+    public function __construct($countryCode, $bank)
+    {
+        parent::__construct();
+        $this->countryCode = $countryCode;
+        $this->bank = $bank;
+    }
+    
+    public function validate($input)
+    {
+        switch (strtolower($this->countryCode)) {
+            case "de":
+                $bav = new BAV();
+                return $bav->isValidBankAccount($this->bank, $input);
+                
+            default:
+                $message = sprintf(
+                        "Cannot validate bank account for country '%s'.",
+                        $this->countryCode);
+                throw new ComponentException($message);
+        }
```

Comment:
The same changes I recommended to [BIC](https://github.com/Respect/Validation/pull/188/files#diff-5) rule.